README.md

```diff
@@ -118,42 +118,42 @@ In the example below, the `inspector` variable will be used.  For the client-sid
 
 ### Validation
 
-* [type](#v_type)
-* [optional](#v_optional)
-* [pattern](#v_pattern)
-* [minLength, maxLength, exactLength](#v_length)
-* [lt, lte, gt, gte, eq, ne](#v_comparators)
-* [someKeys](#v_someKeys)
-* [strict](#v_strict)
-* [exec](#v_exec)
-* [properties](#v_properties)
-* [items](#v_items)
-* [alias](#v_alias)
-* [error](#v_error)
-* [code](#v_code)
+* [type](#type)
+* [optional](#optional)
+* [pattern](#pattern)
+* [minLength, maxLength, exactLength](#length)
+* [lt, lte, gt, gte, eq, ne](#comparators)
+* [someKeys](#someKeys)
+* [strict](#strict)
+* [exec](#exec)
+* [properties](#properties)
+* [items](#items)
+* [alias](#alias)
+* [error](#error)
+* [code](#code)
 
 ### Sanitization
 
-* [type](#s_type)
-* [def](#s_def)
-* [optional](#s_optional)
-* [rules](#s_rules)
-* [min, max](#s_comparators)
-* [minLength, maxLength](#s_length)
-* [strict](#s_strict)
-* [exec](#s_exec)
-* [properties](#s_properties)
-* [items](#s_items)
+* [type](#type-1)
```

Comment:
I tested this by adding `#type-1` in the address bar while browsing the NPM page. It works, but a few others like `def-1` and `optional-1` don't work. I think we need to make the href property of the `a` elements in the `h3` elements under the sanitize section also have "-1" suffixes. Notice that this is the case for the "type" under the sanitize section, which has `type-1`.

Reply:
The problem was that some of the links were not duplicates.

Reply:
Done and tested in d0fadff